ArduCopter/Copter.h

```diff
@@ -473,6 +473,8 @@ class Copter : public AP_Vehicle {
 
     // Used to exit the roll and pitch auto trim function
     uint8_t auto_trim_counter;
+    bool auto_trim_started = false;
+    void auto_trim_cancel();
```

Comment:
a small thing but this breaks the pattern of putting method declarations lower down in the Copter.h categorised (and ordered) by the file they are in.

Reply:
I was grouping it with the related state variables (and managed to resist creating an AutoTrim object to shove stuff into ;-) )

Do you want me to separate the method from related state?

Reply:
yes, please.  the auto_trim() method is already separated from the counter..

Reply:
Oh, duh!  Done :-)